pypesto/profile/validation_intervals.py

```diff
@@ -0,0 +1,106 @@
+import logging
```

Comment:
```suggestion
"""Validation intervals."""

import logging
```
some file level docstring is always good, more informative would be good